module/VuFindSearch/src/VuFindSearch/Command/GetUniqueKeyCommand.php

```diff
@@ -0,0 +1,62 @@
+<?php
+
+/**
+ * Get the backend's unique search key field (currently only supported by Solr)
+ *
+ * PHP version 7
+ *
+ * Copyright (C) Villanova University 2021.
+ *
+ * This program is free software; you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License version 2,
+ * as published by the Free Software Foundation.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program; if not, write to the Free Software
+ * Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301  USA
+ *
+ * @category VuFind
+ * @package  Search
+ * @author   Demian Katz <demian.katz@villanova.edu>
+ * @license  http://opensource.org/licenses/gpl-2.0.php GNU General Public License
+ * @link     https://vufind.org
+ */
+namespace VuFindSearch\Command;
+
+use VuFindSearch\Backend\BackendInterface;
+
+/**
+ * Get the backend's unique search key field (currently only supported by Solr)
+ *
+ * @category VuFind
+ * @package  Search
+ * @author   David Maus <maus@hab.de>
+ * @author   Aleksi Peebles <aleksi.peebles@helsinki.fi>
```

Comment:
Looks like copy pasted authors here?

Reply:
I thought "I'll fix this in my first command, and then copy it to the other two..." except I missed the second instance and then cloned the mistake! ;-) Thanks for catching it, I've just fixed all three instances.